src/api/query/accountInfo.ts

```diff
@@ -0,0 +1,51 @@
+import { bnToBn } from '@polkadot/util';
+import { state } from '@/store/state';
+import { getters } from '@/store/getters';
+import { AccountInfo, Address, Balance } from '../models';
+import { Ref } from 'vue';
+
+export async function subscribeAccountInfo(account: Address, ref: Ref<AccountInfo>) {
+    const api = getters.api(state);
+    if (api === undefined) {
+        throw new Error();
+    }
```

Comment:
These lines are called many times. Could you put it together?

Reply:
Fixed. I moved type check to getter.